cardano-launcher/app/Main.hs

```diff
@@ -32,40 +34,39 @@ import           Data.X509.Extra (failIfReasons, genRSA256KeyPair,
 -- Main
 --------------------------------------------------------------------------------
 
--- | Yes, this is something we probably need to replace with actual loggging.
--- If we want. Otherwise, we can print out to console out just for launcher configs.
-externalDependencies :: ExternalDependencies
-externalDependencies = ExternalDependencies
-    { logInfo            = putTextLn
-    , logError           = putTextLn
-    , logNotice          = putTextLn
-    }
-
 -- | Main function.
 main :: IO ()
 main = do
-
-    let launcherConfig :: LauncherConfig
-        launcherConfig = LauncherConfig
-            { lcfgFilePath    = "./configuration/cert-configuration.yaml"
-            , lcfgKey         = "dev"
-            , lcfgSystemStart = Nothing
-            , lcfgSeed        = Nothing
-            }
+    -- Todo: make it so that you can specify the path via CLI
+    launcherOptions <- either
+        (\err -> throwM $ LauncherOptionsError (show err))
+        return
+        =<< getLauncherOption "./configuration/launcher/launcher-config.demo.yaml"
```

Comment:
Right, so I was planning to introduce that in the next PR

Reply:
@cleverca22 Can you explain why we have `.yaml` configuration here? Didn't we talk about using `dhall` in the configuration?

Reply:
I think using `yaml` file will be the way to go.
This is because daedalus also needs `launch-config.yaml` since it'll be responsible for launching the node and the wallet, but daedalus cannot parse `dhall` files.

You can make it so that the launcher is also responsible for generating the yaml file, but then what if daedalus developers just need the `yaml` file to do something with it..